networkx/algorithms/community/tests/test_asyn_lpa.py

```diff
@@ -87,3 +87,46 @@ def test_several_communities():
     communities = asyn_lpa.asyn_lpa_communities(test)
     result = {frozenset(c) for c in communities}
     assert_equal(result, ground_truth)
+
+def test_two_communities():
+    test = Graph()
+
+    # c1
+    test.add_edge('0', '2')
```

Comment:
Can you do this in a more concise way? For example, something like

```python
c1_edges = [(0, 2), (0, 3), ...]
c2_edges = [...]
G.add_edges_from(c1_edges + c2_edges)
```
Also, you can use integers instead of strings here.